apps/sensenet/src/localization/hungarian.ts

```diff
@@ -82,8 +75,9 @@ const values: DeepPartial<typeof import('./default').default> = {
     logoutCancel: 'Mégsem',
     logoutButtonTitle: 'Kijelentkezés',
     logoutDialogTitle: 'Biztosan kijelentkezel?',
-    loggingOutFrom: "Kijelentkezés a '{0}' repository-ból... ",
-    logoutConfirmText: 'Jelenleg a {0} repository-t használod {1} felhasználóval. Biztosan kijelentkezel?',
+    loggingOutFrom: (repoUrl: string) => `Kijelentkezés a ${repoUrl} repository-ból...`,
```

Comment:
Some of the string resources added to the default.ts are completely missing from the hungarian.ts